pkg/workloadapi/workload.proto

```diff
@@ -74,8 +74,22 @@ message Service {
   repeated string subject_alt_names = 6;
 }
 
+// Workload represents a workload - an endpoint (or collection behind a hostname).
+// The xds primary key is "uid" as defined on the workload below.
+// Secondary (alias) keys are the unique `network/IP` pairs that the workload can be reached at.
```

Comment:
I think we need to actually handle the alias by putting them all in the discovery.Resource object. ztunnel also needs to handle this, so if it requests "ip1" and gets back "Key=uid,aliases=[ip1]" it fetches that

Reply:
@howardjohn I added aliases to the discovery.Resource object returned in `pilot/pkg/xds/workload.go`